builder/tests/builder/src/main/java/io/helidon/builder/test/testsubjects/TestNullable.java

```diff
@@ -1,35 +0,0 @@
-/*
- * Copyright (c) 2022 Oracle and/or its affiliates.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package io.helidon.builder.test.testsubjects;
-
-import io.helidon.builder.Builder;
-
-/**
- * Overrides builder default to allow for null attributes.
- */
-@Builder(allowNulls = true)
-@FunctionalInterface
-public interface TestNullable {
```

Comment:
where is the test for this? We shouldn't project our norms on users who have a different stance on nullability.

Reply:
This is not a generic solution for end users. It is an MVP for Helidon